scripts/messages.js

```diff
@@ -54,15 +59,20 @@ let formatMessage = (message, callback) => {
     let assignees = formatting.getDataMask(message.text, /@[^*\s]+/)
     let name = formatting.getName(message.user.name)
     let msg = formatting.removeFromMessage(message.text, action)
-
+    let blocker = formatting.getStandup(message.text,action, /blockers:(.*)last24/)
```

Comment:
this answers my questions from above:
* will not work if `+standup` is in the middle or end
* will not work if the order is different. 

We should either change approach or say this in README

Reply:
We could even grab the whole standup and paste it in one cell called "standup" instead of separating them into 3 different. Think for now this would be the better solution

Reply:
The reason we should go with the second approach is that it will be a pain to explain other, non-tech users and make them memorize the order.

Reply:
unless there is an easy way to detect "last24", "next24" etc...

Reply:
@zelima very good review and good suggestions, explanation is also super clear